src/objects.rs

```diff
@@ -26,10 +26,16 @@ impl Tags {
     pub fn new() -> Tags {
         Tags(::flat_map::FlatMap::new())
     }
+
     /// Returns if it contains a tag with the given `key` and `value`.
     pub fn contains(&self, key: &str, value: &str) -> bool {
         self.0.get(key).map_or(false, |v| v.as_str() == value)
     }
+
+    /// Consume tags into inner Vec representation.
+    pub fn into_inner(self) -> Vec<(String, String)> {
+        self.0.into_inner()
```

Comment:
Better to output the flat_map. We leak the type in the deref trait, so nothing wrong in returning the flat_map here.

Reply:
I'm ok with that, fixed :)